src/seth/libexec/seth/seth---to-int256

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+### seth---to-int256 -- convert an integer into int256 value
+### Usage: seth --to-int256 <value>
+set -e
+export BC_LINE_LENGTH=5000
```

Comment:
Is the actual max number of characters not `2 * bytes(word_size) = 64`?

Reply:
This is just to force `bc` to not break lines. The actual length is however many digits 2^256 (in decimal) has. But 5000 suffices

Reply:
```
$ BC_LINE_LENGTH=5000 bc <<< 2^256 | wc -c
79
```